testing/test_crossbrowsertesting.py

```diff
@@ -71,7 +71,7 @@ def test_invalid_credentials_env(failure, monkeypatch, tmpdir, username, key):
     monkeypatch.setenv(key, "bar")
     out = failure("--capability", "browser_api_name", "FF46")
     messages = ["missing auth", "basic auth failed"]
-    assert any(message in out for message in messages)
+    assert any(message in out.lower() for message in messages)
```

Comment:
What are we even testing here? The error message from CrossBrowserTesting when creds are bad. Why? The plugin doesn't handle that anyway. 🤔